contrib/build-osx/make_osx

```diff
@@ -15,7 +15,7 @@ cd $src_dir/../..
 export PYTHONHASHSEED=22
 VERSION=`git describe --tags --dirty`
 
-which brew > /dev/null 2>&1 || fail "Please install brew from https://brew.sh/ to continue"
+which brew > /dev/null 2>&1 || which port > /dev/null 2>&1 || fail "Please install either Homebrew (from https://brew.sh/) or MacPorts (from https://www.macports.org/) to continue"
```

Comment:
We call brew directly later in this script so I don't think it works with port installed unless you provide an alternative there too.

Reply:
really? it worked for me!! I don't even have brew installed.. whaaa??

Ok double-checking now.. it totally worked!

Will get back to you in a minute..

Reply:
Yeah ok -- i know why it worked.  Because I already have autoconf, automake, etc installed. So:
```
# brew install autoconf automake
```

Is a no-op on my system (it did say command not found).

Man you guys are persecuting us MacPorts users!!! Ha ha :)

I have 2 options:

1. Modify your script further to cope with macports.. or 
2. just leave you in our brew-centric existance and get rid of that MacPorts thing.

Hmm.  What to do?